src/api/app/controllers/webui/subscriptions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Webui::SubscriptionsController < Webui::WebuiController
+  before_action :require_admin
+
+  def index
+    @subscriptions_form = subscriptions_form
+  end
+
+  def update
+    subscriptions_form.update!(params[:subscriptions])
```

Comment:
It shouldn't go wrong unless the user submits their own hand-crafted http request with invalid parameters. But if it does go wrong whatever reason at least we will know about from errbit rather than having this fail silently.